test/export/StructureDefinition.ResourceExporter.test.ts

```diff
@@ -13,7 +13,7 @@ import {
   ContainsRule,
   FlagRule
 } from '../../src/fshtypes/rules';
-
+import { StructureDefinition } from '../../src/fhirtypes';
```

Comment:
Can you add a space between the imports and the tests?
```suggestion
import { StructureDefinition } from '../../src/fhirtypes';

```